rules-tests/Php74/Rector/Property/TypedPropertyRector/FixtureUnionIntersectionTypes/multiple_types.php.inc

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Rector\Tests\Php74\Rector\Property\TypedPropertyRector\FixtureUnionIntersectionTypes;
+
+final class MultipleTypes
+{
+    /**
+     * @var string|int|float
+     */
+    private $nonNullableVariable;
+    
+    /**
+     * @var string|int|float|null
+     */
+    private $nullableVariable;
+}
+
+?>
+-----
+<?php
+
+namespace Rector\Tests\Php74\Rector\Property\TypedPropertyRector\FixtureUnionIntersectionTypes;
+
+final class MultipleTypes
+{
+    private float|int|string $nonNullableVariable;
+    
+    private float|int|string|null $nullableVariable = null;
```

Comment:
This file is under Tests\Php74

Union types are not supported in PHP 7.4

Shouldn't there be two tests?

-  under Tests\Php74 expected: no changes
-  under Tests\Php80 expected: these changes

Reply:
No, it using config for php 8.0 feature

https://github.com/rectorphp/rector-src/blob/2d9407544477672ce71d4ad8a1dea413dc485b01/rules-tests/Php74/Rector/Property/TypedPropertyRector/config/union_and_intersection_type_rule.php#L15

So the fixture is correct, the expected is when TypedPropertyRector run with php 8.0 feature enabled